src/components/SharedStyledComponents.tsx

```diff
@@ -233,7 +233,7 @@ export const ButtonPrimary = styled(Button)`
   }
 `
 
-export const ButtonSecondary = styled(Button)`
+export const ButtonSecondary = styled(Button)<MarginProps>`
```

Comment:
Missing marginal types such as `mr`

Reply:
Cool, lets add that type to the `Button` component instead of the `ButtonSecondary`.

Reply:
updated!